### PR TITLE
Updated Breakpoints documentation text to match the responsive utilities changes

### PR DESCRIPTION
**Pull request by juliodialpad**
## Description
Needed to add more details to the Dialtone responsive util class page to clarify that all classes are offered in responsive versions
https://switchcomm.atlassian.net/browse/DT-186

## Pull Request Checklist

 - [x] Ask the contributors if a similar effort is already in process or has been solved.
 - [x] Review the [contribution guidelines](https://github.com/dialpad/dialtone/blob/staging/.github/CONTRIBUTING.md).
 - [x] Use `staging` as your pull request's base branch. (All PRs using `production` as its base branch will be declined).
 - [x] Ensure all `gulp` scripts successfully compile.
 - [x] Update, remove, or extend all affected documentation.
 - [x] Ensure no private Dialpad links or info are in the code or pull request description (Dialtone is a public repo!).
 - [x] Request a review from Brad Paugh, David Becher, or Drew Chandler.

### CODE REVIEW

File: docs/utilities/responsive/breakpoints.html [hunk 1]
```diff
@@ -52,11 +52,11 @@
     {% endcodeWellHeader %}
     {% codeWellFooter %}
 {% highlight html linenos %}
-<div>...</div>
-<div class="d-d-none xl:d-d-block">...</div>
-<div class="d-d-none lg:d-d-block">...</div>
-<div class="d-d-none md:d-d-block">...</div>
-<div class="d-d-none sm:d-d-block">...</div>
```
Comment (dialpad-drew) [inline docs/utilities/responsive/breakpoints.html]: The ellipsis is a common pattern we use to not distract from the example. The text itself isn't really important; it's really about what classes are wrapping it.

This is a common thing we do throughout the documentation, too:
* https://deploy-preview-480--dialtone.netlify.app/utilities/borders/radius/
* https://deploy-preview-480--dialtone.netlify.app/utilities/typography/font-family/
* https://deploy-preview-480--dialtone.netlify.app/utilities/typography/color/
* https://deploy-preview-480--dialtone.netlify.app/utilities/borders/color/

I'd probably prefer to keep them in here.

**Review by dialpad-drew — COMMENTED** [review]
The changes themselves look good, though should we be merging this into `staging`? I thought you had a responsive class feature branch?

**Comment by braddialpad** [comment]
> The changes themselves look good, though should we be merging this into `staging`? I thought you had a responsive class feature branch?

I'm guessing he didn't because its just documentation and it doesn't actually depend on the changes from that branch, but yeah we'll have to hold off on merging this until the responsive class stuff is in. merging into the branch might be a little cleaner

File: docs/utilities/responsive/breakpoints.html (hunk 1 above)
Comment (juliodialpad) [reply docs/utilities/responsive/breakpoints.html]: Sure, I'll do that

**Comment by dialpad-drew** [comment]
> > The changes themselves look good, though should we be merging this into `staging`? I thought you had a responsive class feature branch?
> 
> I'm guessing he didn't because its just documentation and it doesn't actually depend on the changes from that branch, but yeah we'll have to hold off on merging this until the responsive class stuff is in. merging into the branch might be a little cleaner

On principle, if documentation represents available code, then doesn't it depend on the changes from that branch?

**Comment by juliodialpad** [comment]
Should I recreate this PR from my responsive feature branch?

**Comment by dialpad-drew** [comment]
> Should I recreate this PR from my responsive feature branch?

I dunno what the diff is, but what happens if you just change the destination branch to your feature branch in this PR?

**Comment by juliodialpad** [comment]
> > Should I recreate this PR from my responsive feature branch?
> 
> I dunno what the diff is, but what happens if you just change the destination branch to your feature branch in this PR?

Nothing happened since there's no difference from that files between branches, I've never tried that before, but it worked

**Review by dialpad-drew — APPROVED** [review]

**Pull request merged**